packages/salesforcedx-sobjects/README.md

```diff
@@ -0,0 +1,5 @@
+# salesforcedx-sobjects
```

Comment:
Probably want to rename this to salesforcedx-sobjects-faux-generator to be more descriptive.